docs/source/getting-started/tabular-data/keys.ipynb

```diff
@@ -0,0 +1,280 @@
+      "text/plain": [
+       "<Figure size 1000x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "context = dp.Context.compositor(\n",
+    "    data=df,\n",
+    "    privacy_unit=dp.unit_of(contributions=36),\n",
+    "    privacy_loss=dp.loss_of(epsilon=1.0),\n",
+    "    split_evenly_over=5,\n",
+    "    margins={\n",
+    "        (\"YEAR\", ): dp.Margin(public_info=\"lengths\",\n",
+    "                              max_partition_length=estimated_max_partition_len, \n",
+    "                              max_partition_contributions=4, \n",
+    "                              max_num_partitions=1),\n",
+    "                    \n",
+    "        (): dp.Margin(public_info= \"lengths\",\n",
+    "                      max_partition_length=estimated_max_partition_len, \n",
+    "                      max_num_partitions=1),\n",
+    "    },\n",
+    ")\n",
+    "\n",
+    "sum_year_dp_public = (\n",
+    "    context.query().group_by(\"YEAR\").agg(\n",
+    "        pl.col(\"HWUSUAL\").fill_null(40.).dp.sum((0,100)).alias(\"DP Sum\")\n",
+    "    ).sort(\"YEAR\")).release().collect()\n",
+    "\n",
+    "plt.figure(figsize=(10, 6))\n",
+    "sns.lineplot(x=sum_year_actual[\"YEAR\"].to_list(), y=sum_year_actual[\"Actual Sum\"].to_list(), marker=\"o\", label=\"Actual\")\n",
+    "sns.lineplot(x=sum_year_dp_public[\"YEAR\"].to_list(), y=sum_year_dp_public[\"DP Sum\"].to_list(), marker=\"o\", label=\"DP: Public\")\n",
+    "sns.lineplot(x=sum_year_dp[\"YEAR\"].to_list(), y=sum_year_dp[\"DP Sum\"].to_list(), marker=\"o\", label=\"DP: Private\")\n",
+    "plt.title('Actual vs DP Sums by Year')\n",
+    "plt.xlabel('Year')\n",
+    "plt.ylabel('Sum')\n",
+    "plt.legend(title='Legend')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The DP values align well with the actual values. Moreover, the DP values computed with private lengths follow a similar trend as the DP values computed with public lengths. \n",
+    "\n",
+    "In this particular example, revealing the partition lengths may not appear consequential. However, sometimes they may indicate the number of individuals in a particular category or be misinterpreted leading to incorrect results.  "
```

Comment:
How could exact partition lengths be misinterpreted?

Reply:
analysts may just aim for higher partition lengths w/ the assumption that more is better? or if the parameter value makes sense in scale, they could assume that it's okayness translates over to other parameters?